src/test/kotlin/com/zepben/evolve/services/network/testdata/stupid/StupidlyLargeNetwork.kt

```diff
@@ -224,35 +224,6 @@ object StupidlyLargeNetwork {
         networkService.connect(createTerminal(networkService, node1, PhaseCode.C, 1), "cn_2")
         networkService.connect(createTerminal(networkService, node1, PhaseCode.B, 2), "cn_2")
 
-        val multiTerminalAcLineSegment1 = AcLineSegment("multiTerminalAcLineSegment1").apply { name = "multiTerminalAcLineSegment1"; assetInfo = oh2 }
-        val multiTerminalAcLineSegment2 = AcLineSegment("multiTerminalAcLineSegment2").apply { assetInfo = oh2; length = Double.NaN }
-        val multiTerminalNode1 = Junction("multiTerminalNode1").apply { name = "multiTerminalNode1" }
-        val multiTerminalNode2 = Junction("multiTerminalNode2").apply { name = "multiTerminalNode2" }
-        val multiTerminalNode3 = Junction("multiTerminalNode3").apply { name = "multiTerminalNode3" }
-
-        networkService.connect(
-            createTerminal(networkService, multiTerminalAcLineSegment1, PhaseCode.A, 1),
-            createTerminal(networkService, multiTerminalAcLineSegment2, PhaseCode.A, 1)
-        )
-        networkService.connect(
-            createTerminal(networkService, multiTerminalAcLineSegment1, PhaseCode.A, 2),
-            createTerminal(networkService, multiTerminalNode1, PhaseCode.A, 1)
-        )
-        networkService.connect(
-            createTerminal(networkService, multiTerminalAcLineSegment1, PhaseCode.A, 3),
-            createTerminal(networkService, multiTerminalNode2, PhaseCode.A, 1)
-        )
-        networkService.connect(
-            createTerminal(networkService, multiTerminalAcLineSegment1, PhaseCode.A, 4),
-            createTerminal(networkService, multiTerminalNode3, PhaseCode.A, 1)
-        )
-
-        networkService.add(multiTerminalAcLineSegment1)
-        networkService.add(multiTerminalAcLineSegment2)
-        networkService.add(multiTerminalNode1)
-        networkService.add(multiTerminalNode2)
-        networkService.add(multiTerminalNode3)
-
```

Comment:
Removing this didn't break any tests, so I'm guessing whatever used to rely on this is either obsolete/removed or has had its own specific tests written.